api/src/passage/management/commands/passage_zwaar_verkeer_hour_aggregation_v2.py

```diff
@@ -118,10 +120,7 @@ def _get_aggregation_query(self, run_date):
                     p.rijrichting = h.rijrichting
         WHERE p.passage_at >= '{run_date}'
         AND p.passage_at < '{run_date + timedelta(days=1)}'
-		AND (
-		    (p.voertuig_soort = 'Bedrijfsauto' AND p.toegestane_maximum_massa_voertuig > 3500) OR 
-		    p.toegestane_maximum_massa_voertuig > 7500
-        )
```

Comment:
Deze wil ik intact houden, dit is waar de privacy officer goedkeuring voor heeft gegeven.